src/sentry/api/endpoints/organization_code_mapping_details.py

```diff
@@ -64,5 +65,11 @@ def delete(self, request, config_id, organization, config):
 
         :auth: required
         """
-        config.delete()
-        return self.respond(status=status.HTTP_204_NO_CONTENT)
+        try:
+            config.delete()
+            return self.respond(status=status.HTTP_204_NO_CONTENT)
+        except ProtectedError:
+            return self.respond(
+                "Cannot delete Code Mapping. There is a Code Owners that depends on it.",
```

Comment:
how do you feel about something that is more direct about what they have to do, something like:

`"Cannot delete Code Mapping. Must delete Code Owner that uses this mapping first."`

Also can we add a quick endpoint test for this response?